### PR TITLE
DOCSUP-3122: Document two expressions ALTER

### PR DESCRIPTION
**Pull request by AnaUvarova**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #14742

### CODE REVIEW

File: docs/en/sql-reference/statements/alter/column.md [hunk 1]
```diff
@@ -122,6 +122,39 @@ Example:
...
+```
+
+```sql
+SELECT column_ttl FROM table_with_ttl;
+```
+
+Result
+```
+As a result after removing the TTL-property from the specified column, you will see full list with removed after TTL functioning lines.
+```
```
Comment (BayoNet) [inline docs/en/sql-reference/statements/alter/column.md:150]: It looks like these lines are not required for the description.

File: docs/en/sql-reference/statements/alter/ttl.md [hunk 2]
```diff
@@ -3,10 +3,57 @@ toc_priority: 44
...
 
 You can change [table TTL](../../../engines/table-engines/mergetree-family/mergetree.md#mergetree-table-ttl) with a request of the following form:
 
 ``` sql
-ALTER TABLE table-name MODIFY TTL ttl-expression
+ALTER TABLE table_name MODIFY TTL ttl_expression;
+```
+
+## REMOVE TTL {remove-ttl}
+
```
Comment (olgarev) [inline docs/en/sql-reference/statements/alter/ttl.md:17]: ## REMOVE TTL {#remove-ttl}

# missed in anchor

File: docs/en/sql-reference/statements/alter/column.md [hunk 3]
```diff
@@ -122,6 +122,26 @@ Example:
 ALTER TABLE visits MODIFY COLUMN browser Array(String)
 ```
 
+## MODIFY REMOVE
+
```
Comment (olgarev) [inline docs/en/sql-reference/statements/alter/column.md:126]: Missed anchor here

File: docs/ru/sql-reference/statements/alter/column.md [hunk 4]
```diff
@@ -110,6 +110,25 @@ MODIFY COLUMN [IF EXISTS] name [type] [default_expr] [TTL]
 ``` sql
 ALTER TABLE visits MODIFY COLUMN browser Array(String)
 ```
+## MODIFY REMOVE
+
```
Comment (olgarev) [inline docs/ru/sql-reference/statements/alter/column.md:114]: Missed anchor here

File: docs/ru/sql-reference/statements/alter/ttl.md [hunk 5]
```diff
@@ -5,10 +5,56 @@ toc_title: TTL
...
+## MODIFY TTL {#modify-ttl}
+
 Вы можете изменить [TTL для таблицы](../../../engines/table-engines/mergetree-family/mergetree.md#mergetree-column-ttl) запросом следующего вида:
 
 ``` sql
 ALTER TABLE table-name MODIFY TTL ttl-expression
 ```
 
+## REMOVE TTL {remove-ttl}
+
```
Comment (olgarev) [inline docs/ru/sql-reference/statements/alter/ttl.md:17]: "#" missed in anchor. Should be {#remove-ttl}

File: docs/en/sql-reference/statements/alter/ttl.md [hunk 6]
```diff
@@ -3,10 +3,57 @@ toc_priority: 44
...
+```
+
+**Example**
+
+Requests and results:
+
+To start the background cleaning using TTL, make this:
+
+```sql
+OPTIMIZE TABLE table_with_ttl FINAL;
```
Comment (alesapin) [inline docs/en/sql-reference/statements/alter/ttl.md:33]: No table definition and now info about data. How reader can guess it?

File: docs/en/sql-reference/statements/alter/ttl.md [hunk 7]
```diff
@@ -3,10 +3,57 @@ toc_priority: 44
...
+ALTER TABLE table_with_ttl REMOVE TTL;
+INSERT INTO table_with_ttl VALUES (now() - INTERVAL 4 MONTH, 2, 'username2');
+OPTIMIZE TABLE table_with_ttl FINAL;
+SELECT * FROM table_with_ttl;
 ```
+
+And now we have nothing to delete.
+
+```text
+--2020-12-11 12:44:57    1       username1
```
Comment (alesapin) [inline docs/en/sql-reference/statements/alter/ttl.md:52]: Comments `--`?

File: docs/en/sql-reference/statements/alter/ttl.md [hunk 8]
```diff
@@ -3,10 +3,57 @@ toc_priority: 44
...
+To start the background cleaning using TTL, make this:
+
+```sql
+OPTIMIZE TABLE table_with_ttl FINAL;
+SELECT * FROM table_with_ttl;
+```
+As a result you see that the second line was deleted.
+
+```text
+2020-12-11 12:44:57    1       username1
```
Comment (alesapin) [inline docs/en/sql-reference/statements/alter/ttl.md:39]: Need formatted output, not tab separated.

File: docs/en/sql-reference/statements/alter/ttl.md [hunk 9]
```diff
@@ -3,10 +3,57 @@ toc_priority: 44
...
+```
+
+```sql
+ALTER TABLE table_with_ttl REMOVE TTL;
+INSERT INTO table_with_ttl VALUES (now() - INTERVAL 4 MONTH, 2, 'username2');
+OPTIMIZE TABLE table_with_ttl FINAL;
+SELECT * FROM table_with_ttl;
 ```
+
+And now we have nothing to delete.
```
Comment (alesapin) [inline docs/en/sql-reference/statements/alter/ttl.md:49]: Now TTL removed and nothing will be deleted.

File: docs/en/sql-reference/statements/alter/ttl.md [hunk 10]
```diff
@@ -3,10 +3,57 @@ toc_priority: 44
...
+
+```sql
+ALTER TABLE table_name MODIFY column_name REMOVE TTL 
+```
+
+**Example**
+
+Requests and results:
+
+To start the background cleaning using TTL, make this:
```
Comment (alesapin) [inline docs/en/sql-reference/statements/alter/ttl.md:30]: Trigger TTL forcefully with OPTIMIZE query.

**Review by alesapin — CHANGES_REQUESTED** [review]

File: docs/en/sql-reference/statements/alter/ttl.md [hunk 11]
```diff
@@ -3,10 +3,76 @@ toc_priority: 44
...
+## REMOVE TTL {#remove-ttl}
+
+Removes TTL-property from the specified column.
+
+Syntax:
+
+```sql
+ALTER TABLE table_name MODIFY column_name REMOVE TTL 
+```
+
```
Comment (olgarev) [inline docs/en/sql-reference/statements/alter/ttl.md:25]: missed ; at the end

**Comment by AnaUvarova** [comment]
@alesapin 
Hello, Alexander, please take a look.

**Review by alesapin — APPROVED** [review]

**Pull request merged**